docs/architecture-overview.md

```diff
@@ -0,0 +1,173 @@
+# Architecture CWA Backend
+
+This document outlines the CWA back end architecture on high level. This document
+does not necessarily reflect the current implementation status in this repository, as there are
+still frequent changes to the exposure notification API by Google/Apple.
+
+## Overview
+
+The Corona Warn App ("CWA") server runs in a Kubernetes cluster on the Open Shift (“OSP”) platform.
+Its main objective is to enable users to take part in the exposure notification framework based
+on Apple/Google specifications. Although CWA aims at compliance to the spec on a protocol level, it
+does not mean automatically that all features will be implemented. Main driver for these assessments is data privacy and protection concerns.
+
+Find the latest specifications of Google/Apple here:
+- [Google Spec](https://static.googleusercontent.com/media/www.google.com/en//covid19/exposurenotifications/pdfs/Exposure-Key-File-Format-and-Verification.pdf)
+- [Apple Spec](https://developer.apple.com/documentation/exposurenotification/setting_up_an_exposure_notification_server?changes=latest_beta)
+
+On a high level, the application consists of two main parts, as shown below.
+
+![Overview Diagram](./images/v4.png)
+
+1. CWA Backend: Handles submission & aggregation/distribution of diagnosis keys and configuration files.
+2. Verification Backend: Deals with test result verification and issues TANs. This back end is managed & deployed
+separately. The repository for those components will be opened up to the community soon.
+
+This document outlines the CWA back end components, which are part of this repository. For the full architectural
+overview, check out the [solution architecture](https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md).
+
+
+## Integration with Other Systems
+
+### Object Store
+
+All mobile app relevant files will be stored on the S3. Those files are:
+-	aggregated files containing the diagnostic keys, which were reported in a specific interval (e.g. hourly)
+-	Daily aggregated files containing the exposure keys, which were reported for this day.
+-	Configuration files, containing the [exposure configuration](https://developer.apple.com/documentation/exposurenotification/enexposureconfiguration) and CWA mobile app config
+-	Additional files regarding meta information of available files/structures/etc.
+
+The files will be pushed to an S3 compliant object store whenever new files become available. File structure definitions for those files can be found in the respective Services chapter.
+
+The mobile application will use a CDN for fetching files, which mirrors all files as a transparent proxy present in the object store.
+
+### Verification Backend
+
+Note: The verification back end design is not yet available as discussions are ongoing.
+
+The verification back end supports the users' journey beginning at a them scanning their QR code printed
```

Comment:
There seems to be some wording mistakes here

Reply:
Thank you for pointing this out